go/master/etcd_client.go

```diff
@@ -27,7 +27,7 @@ type EtcdClient struct {
 	lock      *concurrency.Mutex
 }
 
-// NewEtcdClient creates a new EtcdClient.
+// NewEtcdClient creates a new EtcdClient
```

Comment:
> Subsequent sentences and/or paragraphs can give more details. Sentences should be properly punctuated.

https://github.com/golang/go/wiki/Comments

The comment should be a sentence (ends with ".").